Mathlib/Topology/Maps.lean

```diff
@@ -538,6 +538,10 @@ theorem isClosedMap_iff_closure_image [TopologicalSpace α] [TopologicalSpace β
         _ = f '' c := by rw [hc.closure_eq]⟩
 #align is_closed_map_iff_closure_image isClosedMap_iff_closure_image
 
+/-- A map `f : X → Y` is closed if and only if for all sets `s`, any cluster point of `f '' s` is
+the image by `f` of some cluster point of `s`.
+If you require this for all filters instead of just principal filters, and also that `f` is
+continuous, you get the notion of **proper map**. See `isProperMap_iff_clusterPt`. -/
```

Comment:
For the first time in my life, I think I now understand what a proper map is.

Reply:
Happy to see that I'm not the only one to see this as very satisfying :smiley: